packages/loot-core/migrations/1712784523000_unhide_input_group.sql

```diff
@@ -0,0 +1,8 @@
+BEGIN TRANSACTION;
+
+UPDATE category_groups
+SET
+  hidden = 0
+WHERE is_income;
```

Comment:
Should this be is_income = 1?

Reply:
Wouldn't hurt. Though I tested it (with a hidden non-income category) and it seemed to work as expected this way.